examples/config/outputs.tf

```diff
@@ -0,0 +1,24 @@
+output "api_integrations" {
+  value = module.opsgenie_config.api_integrations
```

Comment:
Not sure if that's the best way to expose outputs for this particular module.
I didn't do it as api keys should be sensitive outputs.

Reply:
Never mind I see that it has changed in next PR
https://github.com/cloudposse/terraform-opsgenie-incident-management/pull/6/files